coeus-webapp/src/main/resources/META-INF/mime.types

```diff
@@ -0,0 +1,765 @@
+application/andrew-inset                        ez
```

Comment:
Hi Terry, 

The zip upload code uses a [MimetypesFileTypeMap](http://docs.oracle.com/javase/7/docs/api/javax/activation/MimetypesFileTypeMap.html) to get the content type of the entries in the uploaded zip, which in turn looks for META-INF/mime.types (among other locations) for the actual file extension / content type mappings. There's probably a better way to do this, but we needed it because without specifying the mime.types a lot of entries were getting categorized as "text/plain" or "application/octet-stream" when they should have been something more specific.